Resources/Prototypes/_LostParadise/Entities/Objects/Decoration/halloween.yml

```
@@ -0,0 +1,34 @@
+- type: entity
+  parent: BaseSign
+  id: LPPPSpiderDeco
+  name: spiders
+  description: Fake spiders that look like they're ready to bite you.
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)